coreneuron/nrnmpi/nrnmpiuse.h

```diff
@@ -35,6 +35,11 @@ THE POSSIBILITY OF SUCH DAMAGE.
   #define NRNMPI 1
 #endif
 
+/* define to 1 if want multisend spike exchange available */
+#ifndef NRN_MULTISEND
```

Comment:
If one knows that multisend will not be used, then it is reasonable to disable it. Otherwise
it can be useful to compare performance. It may be very much faster, especially if there are
few connections per cell or number of connections per cell << number of ranks.